app/app.js

```diff
@@ -22,6 +22,7 @@ var App = Ember.Application.extend(Ember.Evented, {
     }
 
     this.on('user:signed_in', function(user) {
+      $('body').addClass('signed-in');
```

Comment:
It’s my impression that using jQuery to add/remove classes is counter to “the Ember Way”, because you can use a more declarative approach, tying the existence of the class to whatever data you‘re checking. Then there’s no need to manually add/remove classes, they’re just explicitly connected to the data model.

I’ve seen such uses of jQuery throughout the codebase though, as I’m adding acceptance tests, so it’s not a departure from the existing style. But eventually I’d like to move away from this pattern.

Just a note, not something I think needs to change in this PR!